src/zenhub.py

```diff
@@ -69,9 +79,107 @@ def _get_repo_id(repo_name):
         try:
             if repo_name == 'azul':
                 return str(repo['AZUL'])
+            elif repo_name == 'sync-agile-boards':
+                return str(repo['SYNC'])
+            elif repo_name == 'sync-test':
+                return str(repo['SYNCTest'])
         except ValueError as err:
             logger.info(f'{repo_name} is not a known repo')
 
+    def _update_issue_points(self, value):
+        """
+        Change the point estimate for the issue.
+
+        :param int value: The desired value for the point estimate.
+        """
+        logger.debug(f'Changing the current value of story points to {value}')
+
+        url = os.path.join(self.url, 'estimate')
+        json_dict = {'estimate': value}
+        response = requests.put(url, headers=self.headers, json=json_dict)
+
+        if response.status_code == 200:
+            logger.debug(f'Success. {self.issue} now has a story points value of {value}')
+        else:
+            logger.debug(f'Failed to change the story point value of {self.issue} to {value}')
+
+    def _update_issue_pipeline(self, pipeline, pos=None):
+        """
+        Change the pipeline of an issue.
+
+        See https://github.com/ZenHubIO/API#move-an-issue-between-pipelines for further documentation.
+
+        :param str pipeline: A valid string representing a pipeline in Zenhub ('New Issue', 'Icebox'...)
+                             See the Product Development Pipelines section of the following ink for valid values:
+                                 https://www.zenhub.com/blog/how-the-zenhub-team-uses-zenhub-boards-on-github/
+        :param pos: Either 'top', 'bottom', or a 0-based position in the array of tickets in this pipeline.
+        """
+        if pipeline in self.pipeline_ids:
+            logger.debug(f'Changing the current value of pipeline to {pipeline}')
+
+            url = os.path.join(self.url, 'moves')
+            json_dict = {'pipeline_id': self.pipeline_ids[pipeline], 'position': pos or 'top'}
+
+            response = requests.post(url, headers=self.headers, json=json_dict)
+
+            if response.status_code == 200:
+                logger.debug(f'Success. {self.issue} was moved to {pipeline}')
+            else:
+                logger.debug(f'Failed to move {self.issue} to {pipeline}')
+        else:
+            logger.error(f'{pipeline} is not a valid pipeline.')
```

Comment:
We should return the status code and `.reason` of the response. But make sure response to happy path and failure have same structure (e.g., a dict)